models/Loans.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+
+let loanSchema = new mongoose.Schema({ 
+    users_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true,
+        trim: true,
+        minlength: 1,
+        maxlength: 11
+    },
+    guarantor_id: {
+        type: int,
```

Comment:
Cannot be integer. 
Create a ref link to the Guarantor model
Refer to this 
https://alexanderzeitler.com/articles/mongoose-referencing-schema-in-properties-and-arrays/